packages/twenty-server/src/engine/seeder/seeder.service.ts

```diff
@@ -64,98 +65,155 @@ export class SeederService {
+            if (!isDefined(compositeFieldTypeDefinition)) {
+              throw new Error(
+                `Composite field type definition not found for ${field.type}`,
+              );
+            }
+
+            const fieldNames = compositeFieldTypeDefinition.properties
+              ?.map((property) => property.name)
+              .filter(isDefined);
+
+            for (const subFieldName of fieldNames) {
+              const subFieldValue = recordSeed?.[field.name]?.[subFieldName];
+
+              const subFieldValueAsSQLValue =
+                this.turnCompositeSubFieldValueAsSQLValue(
+                  field.type,
+                  subFieldName,
+                  subFieldValue,
+                );
+
+              const subFieldNameAsSQLColumnName = `${field.name}${capitalize(subFieldName)}`;
+
+              objectRecordSeedsAsSQLFlattenedSeeds[
+                subFieldNameAsSQLColumnName
+              ] = subFieldValueAsSQLValue;
+            }
+          } else {
+            const fieldValue = recordSeed[field.name];
+
+            const fieldValueAsSQLValue = this.turnFieldValueAsSQLValue(
+              field.type,
+              fieldValue,
             );
-          }
 
-          const fieldNames = compositeFieldTypeDefinition.properties
-            ?.map((property) => property.name)
-            .filter(isDefined);
-
-          for (const subFieldName of fieldNames) {
-            flattenedSeed[`${field.name}${capitalize(subFieldName)}`] =
-              seed?.[field.name]?.[subFieldName];
+            objectRecordSeedsAsSQLFlattenedSeeds[field.name] =
+              fieldValueAsSQLValue;
           }
-        } else {
-          flattenedSeed[field.name] = seed[field.name];
         }
-      }
 
-      return flattenedSeed;
-    });
+        return objectRecordSeedsAsSQLFlattenedSeeds;
+      },
+    );
+
+    const fieldMetadataNamesAsFlattenedSQLColumnNames = Object.keys(
+      objectRecordSeedsAsSQLFlattenedSeeds[0],
+    );
+
+    const sqlFieldNames = [
+      ...fieldMetadataNamesAsFlattenedSQLColumnNames,
+      'position',
+      'createdBySource',
+      'createdByWorkspaceMemberId',
+      'createdByName',
+    ];
+
+    const sqlValues = objectRecordSeedsAsSQLFlattenedSeeds.map(
+      (flattenedSeed, index) => ({
+        ...flattenedSeed,
+        position: index,
+        createdBySource: 'MANUAL',
+        createdByWorkspaceMemberId: DEV_SEED_WORKSPACE_MEMBER_IDS.TIM,
+        createdByName: 'Tim Apple',
+      }),
+    );
 
     await entityManager
       .createQueryBuilder()
       .insert()
-      .into(`${schemaName}._${objectMetadataAfterFieldCreation.nameSingular}`, [
-        ...fieldMetadataMap,
-        'position',
-      ])
-      .orIgnore()
-      .values(
-        flattenedSeeds.map((flattenedSeed, index) => ({
-          ...flattenedSeed,
-          position: index,
-        })),
+      .into(
+        `${schemaName}._${objectMetadataAfterFieldCreation.nameSingular}`,
```

Comment:
Can you use computeTableName util instead of manually adding "_"? So we keep that logic at the same place in case we change our mind